### ISSUE TITLE
Navigation Bar - Toggle button

### ISSUE THREAD

**Issue by zayed789** [zayed789/Project--Apt-Programmer#1]
The toggle button does not open up when I click on it.
You can refer the code in index.html, Apt Programmer.css and navbar.js and fix the problem.
Project--Apt-Programmer

**Comment by zayed789** [comment]
@Developer404

Issue: The toggle button does not open up when I click on it.
Hey Shaik, this is Zayed. Can you please fix this bug.The toggle button does not open up when I click on it.
I used media queries at max-width: 650px and the toogle button does show up but does not function.
You can view the following files in my repository: Apt-Programmer: 

--> index.html
-->Apt Programmer.css
-->navbar.js

Please help.

**Comment by Developer404** [comment]
bro I wish I could help you, have u tried doing a google search?

**Comment by zayed789** [comment]
@Developer404 
Yes, I did and tried all sorts of things and found out that my VS Code was not able to run the JS file, then I updated it and now it is working. 
Thanks.
